baselines/commoncontrols.md

```diff
@@ -1182,3 +1183,37 @@ To configure additional services per the policy:
 2.	Navigate to **Security** -> **Authentication** -> **Multi-party approval settings**.
 3.	Ensure **Require multi party approval for sensitive admin actions** is checked.
 4.	Click **Save**.
+
+## 18. Early Access Apps
+This section covers whether users have access to early access apps or not.
+
+### Policy
+
+#### GWS.COMMONCONTROLS.18.1v0.2
+Early Access Apps Service Status SHOULD be set to OFF for everyone.
+
+- _Rationale_: Allowing access to early access apps could provide access to apps that are not needed which then creates a security risk. Disabling access to these apps mitigates that risk.
```

Comment:
While I agree that there's a risk here, I don't think this rationale quite covers it. How about:
```suggestion
- _Rationale_: Allowing early access to apps may expose users to apps that have not yet been fully vetted and may still need to undergo robust testing to ensure the latest security standards are met.
```